### PR TITLE
Implements orderable fish food.

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

Most stations now have aquarium kits ready in service, I thought this was super fucking neat but realized 'oh no food, fishy die quickly'. To amend that I've added an orderable fish food crate to synergize with the free aquarium kit. 

## Why It's Good For The Game

Ordering a whole now aquarium kit for the fish food is silly.

## Changelog

:cl:
qol: Fishfood is now more easily acquired through cargo
/:cl:

### CODE REVIEW

**Comment by carshalash** [comment]
Tagged as QoL since I feel like not including both was an oversight.

File: code/modules/cargo/packs/service.dm [hunk 1]
```diff
@@ -280,6 +280,15 @@
 	crate_type = /obj/structure/closet/crate/large
 	discountable = SUPPLY_PACK_UNCOMMON_DISCOUNTABLE
 
+/datum/supply_pack/service/fish_feed
```
Comment (Ghommie) [inline code/modules/cargo/packs/service.dm:283]: Make this a goodie, like almost every fishing-related pack.

Comment (carshalash) [reply code/modules/cargo/packs/service.dm:283]: Wait really? Man, I was gone for too long.

Comment (Ghommie) [reply code/modules/cargo/packs/service.dm:283]: Ok, now that the goodies pack has been added, can you remove this one? I don't think it's a good idea to have two of the same pack in different categories, also people will mostly ignore this one.

**Pull request merged**